### PR TITLE
[SMALLFIX] Do not skip loading metadata if we need to load the direct children

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

**Comment by peisun1115** [comment]
@apc999

File: core/server/src/main/java/alluxio/master/file/FileSystemMaster.java [hunk 1]
```diff
@@ -1900,7 +1900,7 @@ private long loadDirectoryMetadataAndJournal(LockedInodePath inodePath,
    */
   private long loadMetadataIfNotExistAndJournal(LockedInodePath inodePath,
       LoadMetadataOptions options) {
-    if (!inodePath.fullPathExists()) {
+    if (!inodePath.fullPathExists() || options.isLoadDirectChildren()) {
```
Comment (apc999) [inline core/server/src/main/java/alluxio/master/file/FileSystemMaster.java]: could you update the javadoc?

Comment (peisun1115) [inline core/server/src/main/java/alluxio/master/file/FileSystemMaster.java]: done

**Comment by peisun1115** [comment]
@apc999  updated the comment

**Comment by apc999** [comment]
LGTM, thanks for the fix

**Pull request merged**